config/reference_paths.config

```
@@ -43,6 +43,12 @@ params {
   validation_markers_file = "${projectDir}/references/validation-markers.tsv"
   validation_palette_file = "${projectDir}/templates/palettes/validation-group-palette.tsv"
 
+  // TODO: These files will be added to the references dir once we have tagged versions. For now we use URLs.
```

Just clarifying which reference directory (since we also have references on S3 and I was thinking about there first)
```suggestion
  // TODO: These files will be added to the `${projectDir}/references` dir once we have tagged versions. For now we use URLs.
```